front/app/containers/IdeasShow/IdeaProposedBudget.tsx

```diff
@@ -17,11 +18,15 @@ const Container = styled.div`
 
 interface Props {
   className?: string;
-  formattedBudget: string;
+  proposedBudget: number;
 }
 
-const IdeaProposedBudget = memo<Props>(({ className, formattedBudget }) => {
-  return <Container className={className}>{formattedBudget}</Container>;
+const IdeaProposedBudget = memo<Props>(({ className, proposedBudget }) => {
+  return (
+    <Container className={className}>
+      <FormattedCurrency value={proposedBudget} />
```

Comment:
Used FormattedCurrency instead inside IdeaProposedBudget, so we only use 1 currency tool everywhere and it simplifies the usage of this component a lot (no need for big calculations outside, see IdeasShow component where I took it out).